tracer/src/Datadog.Trace.Coverage.collector/AssemblyProcessor.cs

```diff
@@ -824,6 +824,12 @@ private string CopyRequiredAssemblies(AssemblyDefinition assemblyDefinition, Tra
         {
             try
             {
+                if (_settings.TracerHome is null)
+                {
+                    _logger.Error("CopyRequiredAssemblies: TracerHome is not set. Cannot copy Datadog.Trace assembly.");
+                    return string.Empty;
+                }
```

Comment:
This looks like something that shouldn't be possible, otherwise there's other issues 🤔 Maybe there's refactoring to do here @tonyredondo?

Reply:
Yeah, totally agree, the check is required, but it suggests that refactoring would be useful to avoid the _need_ for this check (because basically this is an error scenario that _shouldn't_ happen 😅) But again, not required for this PR, just something for Tony to consider later 🙂